UPGRADING.md

```diff
@@ -139,6 +139,38 @@ Required wiring:
 - entry in SetGenesisModuleOrder
 - entry in SetExportModuleOrder **before `x/bank`**
 
+## Custom Minting Function in `x/mint`
+
+This release refactors the minting process to support a configurable mint function. The minting logic is now abstracted as a `MintFn` with a default implementation that can be overridden.
```

Comment:
i think we should frame this as a new feature, not necessarily a refactoring. refactor is a scary word
```suggestion
This release introduces the ability to configure a mint function in `x/mint`. The minting logic is now abstracted as a `MintFn` with a default implementation that can be overridden.
```